button/button.css

```diff
@@ -0,0 +1,116 @@
+button,
+a.button {
+  background-color: var(--grey-90-a10);
+  border: 0 none;
+  border-radius: 2px;
+  color: var(--grey-90);
+  display: inline-block;
+  font-family: inherit;
+  font-size: 13px;
+  font-weight: 400;
+  height: 32px;
+  line-height: 32px;
+  min-width: 132px;
+  padding: 0 8px;
+  text-align: center;
+  text-decoration: none;
+  vertical-align: middle;
+  width: auto;
+}
+
+
+/* Primary style */
+
+button.button--primary,
+a.button.button--primary {
```

Comment:
Do we need both `.button` and `.button--primary`? I would figure that the second kinda implies the first...

Reply:
(This might be a philosophical objection I have to BEM, but I also feel like the Firefox front-end team will have a similar objection to adding a ton of classes where one would do. 🙂)

Reply:
The notion (from BEM) is that a "primary" button is a *kind* of button, vs a whole new thing. so you say "hey this is a button", and also "it's the primary kind". I'm open to changing it in the name of ergonomics ("They're more guidelines than rules..."), but it does in practice reduce the risk of namespace conflicts.

Reply:
I'm a big fan of BEM for long term legibility. It's really not terribly longwinded at the end of the day, and it allows for strongly scoped styles.

Reply:
Okay, if everyone else likes it, we can keep the extra classes. 🤷‍♂️🙂